content/handbook/product/ux/learning-and-development/tufts-university-capstone-course.md

```
@@ -0,0 +1,158 @@
+- 私たちが日常的に扱っている概念について、複雑さを単純化する明確な言葉を使用してコミュニケーションスキルを向上させる
+- デザインおよびリサーチコミュニティをサポートすることで、人前に出る経験を増やす。
+
+## キャップストーンコース概要
+
+プログラム期間は 15 週間です。学生は 4〜5 人のグループで参加し、業界スポンサーが設定した Human Factors の問題やデザインチャレンジに取り組みます。
+
+学期を通じて、学生は週次レポートとプレゼンテーション（ピアレビューあり）、論文レビュー、スポンサーミーティング／レビューを含む、プロフェッショナルレベルの作業を提供します。プロジェクト管理と計画と共に、ユーザーリサーチの手法と原則が適用されます。このコースの成果には、学生がクラス、学部、スポンサーに発表する最終プロジェクトビデオ、ポスター、プレゼンテーションを作成することが含まれます。
+
+## プロジェクトの仕組み
+
+毎年秋に、プロジェクトリードは GitLab チームメンバーの[ボランティア](#how-to-volunteer)を募集し、プロジェクトの[提案](#project-proposals)を募ります。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)